# Action goal
geometry_msgs/Pose p_init
geometry_msgs/Pose p_final
bool return_to_start # for doing hobbit
---
# Action result
bool success
traj_opt_msgs/Trajectory traj
duration execution_time # time to execute trajectory
---
# Action feedback